deps/chakrashim/src/jsrtcontextshim.cc

```diff
@@ -450,22 +447,13 @@ bool ContextShim::ExecuteChakraShimJS() {
 }
 
 bool ContextShim::ExecuteChakraDebugShimJS(JsValueRef * chakraDebugObject) {
-  wchar_t buffer[_countof(chakra_debug_native) + 1];
-
-  if (StringConvert::CopyRaw<unsigned char, wchar_t>(chakra_debug_native,
-    _countof(chakra_debug_native),
-    buffer,
-    _countof(chakra_debug_native)) != JsNoError) {
-    return false;
-  }
-
-  // Ensure the buffer is null terminated
-  buffer[_countof(chakra_debug_native)] = L'\0';
-
   JsValueRef getInitFunction;
-  if (JsParseScriptWithAttributes(buffer,
+  JsValueRef url;
+  CHAKRA_VERIFY(JsCreateString("chakra_debug.js", strlen("chakra_debug.js"),
```

Comment:
It looks a template helper for JsCreateString and JsCreatePropertyId that automatically deduce C string literal length would reduce a lot of "strlen".